tests/jobs/spearman_correlation_job/test_integration.py

```diff
@@ -48,24 +29,38 @@ def test_main_entry_point_gen_data(monkeypatch):
     res_config = ResConfig("snake_oil.ert")
 
     ert = EnKFMain(res_config)
-    facade = LibresFacade(ert)
-
-    spearman.spearman_job(facade, 1.0, False)
+    sc.SpearmanCorrelationJob(ert).run(*["-t", "1.0"])
 
     # each call represent a scaling job on a cluster, we expect the snake_oil
     # observations to generate this amount of them
     assert scal_job.call_count == 47, "wrong number of clusters"
 
 
+@pytest.mark.skipif(TEST_DATA_DIR is None, reason="no libres test-data")
+@pytest.mark.usefixtures("setup_tmpdir")
+def test_scaling(monkeypatch):
+    test_data_dir = os.path.join(TEST_DATA_DIR, "local", "snake_oil")
+
+    shutil.copytree(test_data_dir, "test_data")
+    os.chdir(os.path.join("test_data"))
+
+    res_config = ResConfig("snake_oil.ert")
+
+    ert = EnKFMain(res_config)
+    sc.SpearmanCorrelationJob(ert).run(*["-t", "1.0"])
+
+    # FIXME: assertsion please
```

Comment:
.....

Reply:
I had forgot/suppressed this. I’ll add the assertions. 

And you should start using … like god intended

Reply:
I just did not know what to say, the `FIXME` already screamed it loud enough. Can I add empty comments though?

Reply:
I cannot 😭

Reply:
damn m$